CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # PLCrashReporter Change Log
 
+## Version 1.7.2 (Under development)
+
+* Fix `xcodebuild` couldn't find modulemap if PLCrashReporter integrated vid Swift Package Manager.
```

Comment:
vid -> via

Please be more specific, it's not clear what scenarious exactly are affected.